code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -76,14 +76,18 @@ var/bomb_set
 
 	if(panel_open && (istype(O, /obj/item/multitool) || istype(O, /obj/item/wirecutters)))
 		return attack_hand(user)
-
-	if(extended)
-		if(istype(O, /obj/item/disk/nuclear))
+	
+	if(istype(O, /obj/item/disk/nuclear))
+		if(extended)
+			if(O.flags & NODROP)
+				to_chat(user, "<span class='notice'>\The [O] is stuck to your hand!</span>")

```

Comment:
shouldn't you return after the to_chat otherwise it'll still try to run the code after?

Reply:
aye good spots. Completely forgot it.

Reply:
it's better to check for if the user drops the item and returning, rathe than doing it this way; this is kinda icky.

Reply:
changed to unEquip

Reply:
usually this is `!user.drop_item()`

Reply:
changed it